tasks/internal/update-fbc-catalog-task/update-fbc-catalog-task.yaml

```diff
@@ -197,10 +208,39 @@ spec:
         retry 5 kinit -V "${KRB5_PRINCIPAL}" -k -t "/tmp/keytab"
 
         set -x
-        # check if this fbc fragment is opt-in
-        echo "Fetching the image bundle from $(params.fbcFragment)..."
-        PULL_SPEC_LIST=$(opm render "$(params.fbcFragment)" | jq -r \
-        'select(.schema == "olm.bundle") | "\(.image)" | split("@")[0]' |uniq)
+        # Validate that required environment variables are set
+        : "${FBC_FRAGMENTS:?FBC_FRAGMENTS environment variable is required}"
```

Comment:
I had to google to understand this properly. I am not against using it (I like how handy bash is with the built in expressions) but maybe it is beneficial using a longer but more readable expression. Just a consideration, up to you to keep, or change it.

Reply:
I can replace that with 
```
 if [ -z "${FBC_FRAGMENTS:-}" ]; then
      echo "Error: FBC_FRAGMENTS environment variable is required"
      exit 1
  fi
  if [ -z "${FROM_INDEX:-}" ]; then
      echo "Error: FROM_INDEX environment variable is required"
      exit 1
  fi
```